From 94380f7852236273e0cee35eea0bff895f70c3c9 Mon Sep 17 00:00:00 2001
From: "Huang, Tao" <huangtao@rock-chips.com>
Date: Mon, 27 Feb 2017 20:51:48 +0800
Subject: [PATCH] arm64: dts: rockchip: cleanup rk3399-android.dtsi

default enable rkvdec and vpu.
rga is default on, remove duplicate configuration.

Change-Id: I8375b2202a81977238e8120e1c2d60f2130844b5
Signed-off-by: Huang, Tao <huangtao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-android.dtsi     |  8 ++++++++
 .../boot/dts/rockchip/rk3399-box-808-android-6.0.dts |  4 ----
 .../boot/dts/rockchip/rk3399-box-android-6.0.dtsi    |  4 ----
 arch/arm64/boot/dts/rockchip/rk3399-box.dtsi         | 12 ------------
 .../dts/rockchip/rk3399-evb-rev3-android-edp.dts     |  7 -------
 .../boot/dts/rockchip/rk3399-mid-818-android-6.0.dts |  4 ----
 .../boot/dts/rockchip/rk3399-mid-818-android.dts     | 12 ------------
 .../dts/rockchip/rk3399-sapphire-excavator-box.dts   |  8 --------
 .../dts/rockchip/rk3399-sapphire-excavator-edp.dts   |  8 --------
 arch/arm64/boot/dts/rockchip/rk3399-tve1205g.dts     | 12 ------------
 10 files changed, 8 insertions(+), 71 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi
index ecf3e93dd3c0..b83b66566161 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi
@@ -305,10 +305,18 @@
 	#sound-dai-cells = <0>;
 };
 
+&rkvdec {
+	status = "okay";
+};
+
 &usbdrd_dwc3_0 {
 	dr_mode = "otg";
 };
 
+&vpu {
+	status = "okay";
+};
+
 &pinctrl {
 	isp {
 		cif_clkout: cif-clkout {
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-box-808-android-6.0.dts b/arch/arm64/boot/dts/rockchip/rk3399-box-808-android-6.0.dts
index 47cdb21c4d39..e2520019378c 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-box-808-android-6.0.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-box-808-android-6.0.dts
@@ -526,10 +526,6 @@
 	mali-supply = <&vdd_gpu>;
 };
 
-&rga {
-	status = "okay";
-};
-
 &tsadc {
 	/* tshut mode 0:CRU 1:GPIO */
 	rockchip,hw-tshut-mode = <1>;
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-box-android-6.0.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-box-android-6.0.dtsi
index caabb097cacb..29244ab0fa76 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-box-android-6.0.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-box-android-6.0.dtsi
@@ -525,10 +525,6 @@
 	mali-supply = <&vdd_gpu>;
 };
 
-&rga {
-	status = "okay";
-};
-
 &threshold {
 	temperature = <85000>;
 };
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-box.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-box.dtsi
index 3c2ca90d1524..f24d5f00437e 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-box.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-box.dtsi
@@ -519,10 +519,6 @@
 	mali-supply = <&vdd_gpu>;
 };
 
-&rga {
-	status = "okay";
-};
-
 &threshold {
 	temperature = <85000>;
 };
@@ -826,14 +822,6 @@
 	pmu1830-supply = <&vcc_1v8>;
 };
 
-&vpu {
-	status = "okay";
-};
-
-&rkvdec {
-	status = "okay";
-};
-
 &rockchip_suspend {
 	status = "okay";
 	rockchip,sleep-debug-en = <0>;
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-edp.dts b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-edp.dts
index 55ca4d213698..0525be2fb53a 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-edp.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-edp.dts
@@ -103,10 +103,3 @@
 	status = "okay";
 };
 
-&vpu {
-	status = "okay";
-};
-
-&rkvdec {
-	status = "okay";
-};
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android-6.0.dts b/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android-6.0.dts
index 9c234e5af3bf..16053c3bcd73 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android-6.0.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android-6.0.dts
@@ -778,10 +778,6 @@
 	mali-supply = <&vdd_gpu>;
 };
 
-&rga {
-	status = "okay";
-};
-
 &spi1 {
 	status = "disabled";
 	max-freq = <50000000>;
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android.dts b/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android.dts
index 28906815a93c..d7e49795dc96 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android.dts
@@ -770,14 +770,6 @@
 	status = "okay";
 };
 
-&vpu {
-	status = "okay";
-};
-
-&rkvdec {
-	status = "okay";
-};
-
 &cpu_l0 {
 	cpu-supply = <&vdd_cpu_l>;
 };
@@ -807,10 +799,6 @@
 	mali-supply = <&vdd_gpu>;
 };
 
-&rga {
-	status = "okay";
-};
-
 &spi1 {
 	status = "disabled";
 	max-freq = <50000000>;
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-box.dts b/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-box.dts
index 77f0f5d7422d..f76876d79f50 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-box.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-box.dts
@@ -162,11 +162,3 @@
 			<0xb5	KEY_BACKSPACE>;
 	};
 };
-
-&vpu {
-	status = "okay";
-};
-
-&rkvdec {
-	status = "okay";
-};
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp.dts b/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp.dts
index af567a089595..d4d609f3449f 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp.dts
@@ -120,11 +120,3 @@
 &route_edp {
 	status = "okay";
 };
-
-&vpu {
-	status = "okay";
-};
-
-&rkvdec {
-	status = "okay";
-};
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-tve1205g.dts b/arch/arm64/boot/dts/rockchip/rk3399-tve1205g.dts
index a9acf0bcbd28..a87119543322 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-tve1205g.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-tve1205g.dts
@@ -859,19 +859,11 @@
 	status = "okay";
 };
 
-&rkvdec {
-	status = "okay";
-};
-
 &route_edp {
 	status = "okay";
 	logo,mode = "center";
 };
 
-&rga {
-	status = "okay";
-};
-
 &saradc {
 	status = "okay";
 };
@@ -1017,10 +1009,6 @@
 	status = "okay";
 };
 
-&vpu {
-	status = "okay";
-};
-
 &pinctrl {
 	sdio-pwrseq {
 		wifi_enable_h: wifi-enable-h {
-- 
2.35.3

